crates/chia-sdk-driver/src/primitives/clawback.rs

```diff
@@ -22,6 +38,74 @@ pub struct Clawback {
 }
 
 impl Clawback {
+    pub fn parse_children(
+        allocator: &mut Allocator,
+        parent_puzzle: Puzzle, // this could be any puzzle type
+        parent_solution: NodePtr,
+    ) -> Result<Option<Vec<Self>>, DriverError>
+    where
+        Self: Sized,
+    {
+        let output = run_puzzle(allocator, parent_puzzle.ptr(), parent_solution)?;
+        let conditions = Vec::<Condition>::from_clvm(allocator, output)?;
+        let mut outputs = Vec::<Clawback>::new();
+        let mut metadatas = Vec::<ClawbackMetadata>::new();
+        let mut puzhashes = Vec::<[u8; 32]>::with_capacity(conditions.len());
+        for condition in conditions {
+            match condition {
+                Condition::CreateCoin(cc) => puzhashes.push(cc.puzzle_hash.into()),
+                Condition::Remark(rm) => match allocator.sexp(rm.rest) {
```

Comment:
I think you could use clvm-traits here to do it like this:

```rs
let Ok(_, (bytes, _)) = <match_list!(MatchByte<2>, Bytes)>::from_clvm(allocator, rm.rest) else {
    continue;
};

// Parse the metadata from the bytes here
```

Reply:
I think the parsing is too complicated here for this